library/packages/src/lib/y2packager/product.rb

```diff
@@ -65,6 +65,22 @@ def from_h(product)
         Y2Packager::Product.new(params)
       end
 
+      # Create a product from Y2Packager::Resolvable
+      # @param product [Y2Packager::Resolvable] product
+      # @param installation_package [String] installation package name
+      # @param displayorder [Integer] display order from the package provides
+      # @return [Y2Packager::Product] converted product
+      def from_resolvable(product, installation_package = "not defined",
```

Comment:
why "not defined"?

Reply:
Discussed in IRC: we considered using `nil` but @schubi2 was afraid of possible side-effects so we decided to go with an empty string. Good enough IMHO.